WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Split Shipments/SelectableShipmentRow.swift

```diff
@@ -0,0 +1,96 @@
+import Yosemite
+import SwiftUI
+
+/// Row for a selectable shipment item to ship with the Woo Shipping extension.
+struct SelectableShipmentRow: View {
+    @ObservedObject private var viewModel: SelectableShipmentRowViewModel
+
+    init(viewModel: SelectableShipmentRowViewModel) {
+        self.viewModel = viewModel
+    }
+
+    @ScaledMetric private var scale: CGFloat = 1
+
+    var body: some View {
+        AdaptiveStack(spacing: Layout.horizontalSpacing) {
+            if viewModel.isSelectable {
+                selectionCircle(selected: viewModel.selected)
+                    .contentShape(Rectangle())
+                    .onTapGesture {
+                        viewModel.handleTap()
+                    }
+            }
+
+            ProductImageThumbnail(productImageURL: viewModel.item.imageUrl,
```

Comment:
Nit: the thumbnails seem quite blurry compared to those on the order details screen. Please feel free to file a separate GH issue for this as the product selector screen seems to have the same issue.

Reply:
🕵️‍♀️ Logged an issue https://github.com/woocommerce/woocommerce-ios/issues/15401